app/presenters/reservation_calendar.rb

```diff
@@ -0,0 +1,19 @@
+class ReservationCalendar < SimpleDelegator
+
+  def initialize(reservation)
+    super(reservation)
+  end
+
+  def as_ical
+    cal = Icalendar::Calendar.new
+    cal.event do |e|
+      e.dtstart = Icalendar::Values::DateTime.new(reserve_start_at)
+      e.dtend = Icalendar::Values::DateTime.new(reserve_end_at)
+      e.summary = "Reservation for #{product.name}"
```

Comment:
We should probably include the facility name in there. Maybe `"#{facility.abbreviation}: #{product.name}"`? I'm not sure we need "Reservation for". I could be convinced otherwise, though.